accounting-service/accounting-api/build.gradle

```diff
@@ -1,29 +1,17 @@
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.1.4.RELEASE")
-    }
+plugins {
+    id "com.jfrog.artifactory" version "4.9.8"
+    id 'maven-publish'
+    id 'idea'
+    id 'io.spring.dependency-management'
 }
 
-
-apply plugin: 'java'
-apply plugin: 'idea'
-apply plugin: 'io.spring.dependency-management'
-
 group 'com.microservices.accounting.api'
 version = apiVersion
 
 jar {
     archiveBaseName = "accounting-api"
 }
 
-repositories {
```

Comment:
Because repositories are declared once in top of multi-module project. Instead of duplicating things in each sub-project separately. And handled by Artifactory to abuse caching as much as possible.